test/data/basic.ts

```diff
@@ -0,0 +1,26 @@
+import { BATCH_TIME } from 'const'
+import BN from 'bn.js'
+
+export const DATE: Date = new Date('2017-04-23T08:30:15.500Z') // Epoch: 1492936215500 mills
+export const DATE_EPOCH = Math.floor(DATE.valueOf() / 1000) // 1492936215 sec
+export const BATCH_NUMBER = Math.floor(DATE_EPOCH / BATCH_TIME) // Math.floor(1492936215 / 300) = 4976454
+export const BATCH_SECOND = DATE_EPOCH % BATCH_TIME // 1492936215 % 300 = 15
+
+export const USER_1 = '0x1111111111111111111111111111111111111111'
+export const USER_2 = '0x2222222222222222222222222222222222222222'
+export const USER_3 = '0x3333333333333333333333333333333333333333'
+export const USER_4 = '0x4444444444444444444444444444444444444444'
+export const USER_5 = '0x5555555555555555555555555555555555555555'
+
+export const TOKEN_1 = '0x9999999999999999999999999999999999999991'
+export const TOKEN_2 = '0x9999999999999999999999999999999999999992'
+export const TOKEN_3 = '0x9999999999999999999999999999999999999993'
+export const TOKEN_4 = '0x9999999999999999999999999999999999999994'
+export const TOKEN_5 = '0x9999999999999999999999999999999999999995'
+export const TOKEN_6 = '0x9999999999999999999999999999999999999996'
+export const TOKEN_7 = '0x9999999999999999999999999999999999999997'
+
+export const AMOUNT = new BN('1000000000000000000') // 1 Ether
+export const AMOUNT_SMALL = new BN('200000000000000000') // 0.2 Ether
```

Comment:
Suggestion: you could use `toWei` here to avoid any mistakes with extra/missing zeros

Reply:
I did at first, this is why the utils incorporate it. But then I saw that BN only accept integers, and I wanted LOTS of decimals for the examples